polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -861,10 +921,6 @@ mod benches {
 		[pallet_child_bounties, ChildBounties]
 		[pallet_claims, Claims]
 		[pallet_collective, Council]
-		// Fixme #177: For some reason the benchmarks write both to the same pallet_collective weight
```

Comment:
I see no reason to fix this. of course the benchmark is the same for two instances of the same pallet